recipes/pretextmap/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name = "PretextMap" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/wtsi-hpag/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: d8667aba7013ea24548d71795c84dbe6482ac9ff7f77b88552112733ebaed700
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }} # [linux64]
+    - make
+    - clang>=9.0.0
```

Comment:
I assume earlier clang versions don't work, yes?

Reply:
That's right,
I've only tested myself on clang 9 and clang 10, but I know clang 4 is too old

Reply:
Specifically, clang 4 is missing the __get_cpuid_count intrinsic